sphinx_gallery/tests/tinybuild/Makefile

```diff
@@ -1,3 +1,5 @@
+SPHINXOPTS=
```

Comment:
What does this do?

Reply:
Just sets the variable at the top so it's clear that it can be used later as `make html SPHINXOPTS=-vv` for example (I'm actually not sure if it can be used otherwise, but even if it can, it's nice to have the indication at the top of the file of the variables that will/can be used)